bot/cogs/source.py

```diff
@@ -54,15 +54,20 @@ async def source_command(self, ctx: Context, *, source_item: SourceConverter = N
         url = self.get_source_link(source_item)
         await ctx.send(embed=await self.build_embed(url, source_item))
 
-    @staticmethod
-    def get_source_link(source_item: Union[HelpCommand, Command, Cog]) -> str:
+    def get_source_link(self, source_item: Union[HelpCommand, Command, Cog]) -> str:
         """Build GitHub link of source item."""
         if isinstance(source_item, HelpCommand):
             src = type(source_item)
             filename = inspect.getsourcefile(src)
         elif isinstance(source_item, Command):
-            src = source_item.callback.__code__
-            filename = src.co_filename
+            if source_item.cog_name == "Alias":
+                cmd_name = source_item.callback.__name__.replace("_alias", "")
+                cmd = self.bot.get_command(cmd_name.replace("_", " "))
+                src = cmd.callback.__code__
+                filename = src.co_filename
```

Comment:
This is fragile but I think it's a decent solution given what you have to work with. It's more of a problem with the design of aliases than with this code. I did want to look into overhauling aliases (one idea was to use decorators on commands to create aliases), but for now, I think this code is OK.